src/Integration/Telemetry/Payload/CFamilyProjectTypes.cs

```diff
@@ -0,0 +1,33 @@
+﻿/*
+ * SonarLint for Visual Studio
+ * Copyright (C) 2016-2021 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using Newtonsoft.Json;
+
+namespace SonarLint.VisualStudio.Integration.Telemetry.Payload
+{
+    public sealed class CFamilyProjectTypes
```

Comment:
After discussion with Marco yesterday, CMake is the priority. Vcx was not in the MMF so it is nice to have, but he said that if it's low effort he'd want that too.
I'll add that after we're done with cmake if there is nothing else to do.